lua/mason-lspconfig/init.lua

```diff
@@ -24,6 +24,10 @@ function M.setup(config)
         require "mason-lspconfig.ensure_installed"()
     end
 
+    require("mason-registry").register_aliases({
+        ['lua-language-server'] = { 'luals' }
+    })
```

Comment:
Something like this ought to do it:

```suggestion
require("mason-registry").register_aliases(_.map(function(server_name)
    return { server_name }
end, require("mason-lspconfig.mappings.server").package_to_lspconfig))
```

Reply:
Oh true I forgot about that mapping, done